starter-web-netty/src/main/resources/application.yml

```diff
@@ -10,7 +10,7 @@ micronaut:
       configurations:
         web:
           allowedOrigins:
-            - ^http(|s):\/\/micronaut\.io$
+            - ${CORS_ALLOWED_ORIGIN:^http(|s):\/\/micronaut\.io$}
```

Comment:
you need to escape the default value with backticks becase there is a colon:
I think it should be ${CORS_ALLOWED_ORIGIN:`^http(|s):\/\/micronaut\.io$`}

Reply:
Good catch @ilopmar, thanks. Addressed b3971f0